docs/settings.rst

```diff
@@ -1231,6 +1231,12 @@ Following are example ways to specify your preferred theme::
     # Specify a customized theme, via absolute path
     THEME = "/home/myuser/projects/mysite/themes/mycustomtheme"
 
+The built-in ``simple`` can be customized using the following settings.
+
+.. data:: STYLESHEET_URL
+
+   The URL of a theme to use as a style sheet.
```

Comment:
"The URL of the stylesheet to use"

I understand the intent behind the "theme" here but I think it could be confused with `pelican` theme